bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/RowResultAdapter.java

```diff
@@ -97,6 +99,7 @@ boolean isMarkerRow() {
     private List<String> labels;
```

Comment:
Please add javadoc to the class:

<p>This adapter is responsible for cell deduplication. bigtable-hbase will convert a MUST_PASS_ONE filter into an Interleave. Unfortunately there is a bit of a mismatch between the 2 filters: MUST_PASS_ONE will not duplicate the cell if it matches multiple branches of the of MUST_PASS_ONE, but Interleave will. This adapter will pave over the difference by removing the duplicate cells while building the Result. However, HBase's WhileMatchFilter depends on duplicate labelled cells for its implementation. So this adapter will not deduplicate labelled cells.